execution.py

```diff
@@ -1,502 +1,502 @@
-            ILDC().clean_diskpart(_)
-            if int(_) not in pd_ids.keys():
-                msg = 'Disk index '+str(_)+ ' Already used by other process'
-                print(msg)
-                LogCreat().logger_error.error(msg)
-                flag = 1
-            else:
-                if _ in self.config_dict['co disk']:
-                    self.co_disk.append(pd_ids[int(_)])
-                else:
-                    self.disk_pool_disk.append(pd_ids[int(_)])
-        del self.config_dict
-        return flag
-    def input_for_test(self):
-        '''
-        This method pass disk and workload details to the tool.
-        Arguments : None
-        Return: None
-        '''
-        flag_run  = 0
-        vd_name = ''
-        workload = ''
-        self.file = open(r"../../../Config/Test.txt", "r+")
-        data = self.file.readlines()
-        self.file.close()
-        if data == []:
-            msg = 'There is nothing to perform plz give vdbench configuration'
-            LogCreat().logger_info.info(msg)
-        else:
-            virtual_disk = data[0].split()
-            vd_name = virtual_disk[0]
-            workload = virtual_disk[1]
-            flag_run = 1
-            file = open(r"../../../Config/Test.txt", "w+")
-            for _ in data[1:]:
-                file.write(_)
-            file.close()
-        return vd_name, workload, flag_run
-    def execute_test(self):
-        '''
-        This method execute workload and create all required
-        configuration of SSY.
-        Arguments : None
-        Return: None
-        '''
-        vd_name, workload, flag_run  = self.input_for_test()
-        if flag_run == 1:
-            print('************************'\
-                  'Test Started************************\n')
-            LogCreat().logger_info.info('************************'\
-                                        'Test Started************************')
-            time.sleep(10)
-            if vd_name.lower() != "standard":
-                self.test_enable_cap_opt_at_server()
-            time.sleep(15)
-            self.create_diskpool(vd_name)
-            time.sleep(15)
-            self.stop_server()
-            time.sleep(25)
-            self.start_server()
-            time.sleep(25)
-            self.test_create_virtual_disk(vd_name)
-            time.sleep(50)
-            self.set_vd_properties(vd_name)
-            time.sleep(5)
-            self.test_serve_vd_to_host()
-            time.sleep(10)
-            diskindex = self.initialize_vd()
-            time.sleep(5)
-            print('************************'\
-                  'VdBench Execution Started************************\n')
-            LogCreat().logger_info.info('************************'\
-                                        'VdBench Execution Started************************')
-            VdBenchRun().run(vd_name, workload, diskindex)
-            print('Result creation completed')
```

Comment:
Use Logger.info or stdout instead of Print

Reply:
This applies throughout the script

Reply:
Its for user experience.
logging also done.